.travis.yml

```diff
@@ -26,5 +26,4 @@ script:
   # let build fail on Clippy warnings
   # and also check tests and none-default crate features
   - cargo clippy --all-targets --all-features -- -D warnings
-  - cargo build
-  - cargo check
+  - cargo test
```

Comment:
We could also call `./build --ci` and be done with it. But then Travis loses the info about the individual build steps...

Reply:
IDK then we would execute all the useless commands we need for their CI. I would keep travis seperated from this

Reply:
`./build --ci` refers to emulation of travis, not speedcenter

Reply:
bikeshedding!

Reply:
Yes but finding the script_path in travis is not necessary at all. This would just clutter the travis log. Our travis log will probably grow really big sooner or later, so running as few commands as possible and having the info about the steps is a advantage IMO.